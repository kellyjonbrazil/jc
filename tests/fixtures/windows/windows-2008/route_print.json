{
  "interface_list": [
    {
      "interface_index": 11,
      "mac_address": "00:0c:29:76:c8:d0",
      "description": "Intel(R) PRO/1000 MT Network Connection"
    },
    {
      "interface_index": 1,
      "mac_address": null,
      "description": "Software Loopback Interface 1"
    },
    {
      "interface_index": 12,
      "mac_address": null,
      "description": "Microsoft ISATAP Adapter"
    }
  ],
  "ipv4_route_table": {
    "active_routes": [
      {
        "network_destination": "0.0.0.0",
        "netmask": "0.0.0.0",
        "gateway": "192.168.23.1",
        "interface": "192.168.23.154",
        "metric": "266"
      },
      {
        "network_destination": "127.0.0.0",
        "netmask": "255.0.0.0",
        "gateway": "On-link",
        "interface": "127.0.0.1",
        "metric": "306"
      },
      {
        "network_destination": "127.0.0.1",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "127.0.0.1",
        "metric": "306"
      },
      {
        "network_destination": "127.255.255.255",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "127.0.0.1",
        "metric": "306"
      },
      {
        "network_destination": "192.168.23.0",
        "netmask": "255.255.255.0",
        "gateway": "On-link",
        "interface": "192.168.23.154",
        "metric": "266"
      },
      {
        "network_destination": "192.168.23.154",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "192.168.23.154",
        "metric": "266"
      },
      {
        "network_destination": "192.168.23.255",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "192.168.23.154",
        "metric": "266"
      },
      {
        "network_destination": "224.0.0.0",
        "netmask": "240.0.0.0",
        "gateway": "On-link",
        "interface": "127.0.0.1",
        "metric": "306"
      },
      {
        "network_destination": "224.0.0.0",
        "netmask": "240.0.0.0",
        "gateway": "On-link",
        "interface": "192.168.23.154",
        "metric": "266"
      },
      {
        "network_destination": "255.255.255.255",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "127.0.0.1",
        "metric": "306"
      },
      {
        "network_destination": "255.255.255.255",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "192.168.23.154",
        "metric": "266"
      }
    ],
    "persistent_routes": [
      {
        "network_address": "0.0.0.0",
        "netmask": "0.0.0.0",
        "gateway_address": "192.168.23.1",
        "metric": "Default"
      }
    ]
  },
  "ipv6_route_table": {
    "active_routes": [
      {
        "interface": 1,
        "metric": "306",
        "network_destination": "::1/128",
        "gateway": "On-link"
      },
      {
        "interface": 11,
        "metric": "266",
        "network_destination": "fe80::/64",
        "gateway": "On-link"
      },
      {
        "interface": 11,
        "metric": "266",
        "network_destination": "fe80::9595:e8d9:b190:ecac/128",
        "gateway": "On-link"
      },
      {
        "interface": 1,
        "metric": "306",
        "network_destination": "ff00::/8",
        "gateway": "On-link"
      },
      {
        "interface": 11,
        "metric": "266",
        "network_destination": "ff00::/8",
        "gateway": "On-link"
      }
    ],
    "persistent_routes": []
  }
}
