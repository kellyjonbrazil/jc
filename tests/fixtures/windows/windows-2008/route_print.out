===========================================================================
Interface List
 11...00 0c 29 76 c8 d0 ......Intel(R) PRO/1000 MT Network Connection
  1...........................Software Loopback Interface 1
 12...00 00 00 00 00 00 00 e0 Microsoft ISATAP Adapter
===========================================================================

IPv4 Route Table
===========================================================================
Active Routes:
Network Destination        Netmask          Gateway       Interface  Metric
          0.0.0.0          0.0.0.0     192.168.23.1   192.168.23.154    266
        127.0.0.0        255.0.0.0         On-link         127.0.0.1    306
        127.0.0.1  255.255.255.255         On-link         127.0.0.1    306
  127.255.255.255  255.255.255.255         On-link         127.0.0.1    306
     192.168.23.0    255.255.255.0         On-link    192.168.23.154    266
   192.168.23.154  255.255.255.255         On-link    192.168.23.154    266
   192.168.23.255  255.255.255.255         On-link    192.168.23.154    266
        224.0.0.0        240.0.0.0         On-link         127.0.0.1    306
        224.0.0.0        240.0.0.0         On-link    192.168.23.154    266
  255.255.255.255  255.255.255.255         On-link         127.0.0.1    306
  255.255.255.255  255.255.255.255         On-link    192.168.23.154    266
===========================================================================
Persistent Routes:
  Network Address          Netmask  Gateway Address  Metric
          0.0.0.0          0.0.0.0     192.168.23.1  Default
===========================================================================

IPv6 Route Table
===========================================================================
Active Routes:
 If Metric Network Destination      Gateway
  1    306 ::1/128                  On-link
 11    266 fe80::/64                On-link
 11    266 fe80::9595:e8d9:b190:ecac/128
                                    On-link
  1    306 ff00::/8                 On-link
 11    266 ff00::/8                 On-link
===========================================================================
Persistent Routes:
  None